frontend/src/app/patients/PatientForm.jsx

```diff
@@ -160,7 +160,7 @@ const PatientForm = (props) => {
       role: patient.role,
       email: patient.email,
       county: patient.county,
-      race: patient.race,
+      race: patient.race ? Object.keys(patient.race) : null,
```

Comment:
I can't see down far enough into the stack to be confident—is this definitely not going to pick up a box that just got de-selected?